assets/node.yaml

```diff
@@ -57,6 +57,8 @@ spec:
             - csi-driver-shared-resource
           args:
             - --config=/var/run/configmaps/config/config.yaml
+            - --config=/var/run/configmaps/share-config-cm/sharedconfigmap-list.yaml
+            - --config=/var/run/configmaps/share-secret-cm/sharedsecret-list.yaml
```

Comment:
Do these need to have separate command line flags? How will we tell them apart?

Reply:
We want to have all these three locations for configmaps. Confused about placement, Could you please put on suggestions about how to bring all these three into the pods.

Reply:
@gabemontero Could you please share your thoughts on this.

Reply:
you are mounting the predefined set of shared secrets and shared config maps with these two new args, right @prietyc123 ?

if so, it makes sense to me @coreydaley that these are separate files stored in separate locations, and hence we have separate args/flags pointing to those separate locations.

Just brought up the EP and that is what it describes.

If you think we are misinterpreting you comment @coreydaley, can you please elaborate / re-phrase what you are trying to get at?

Reply:
> you are mounting the predefined set of shared secrets and shared config maps with these two new args, right @prietyc123 ?

yes, That's for sharing your thoughts Gabe 👍

Reply:
I probably wasn't clear enough in my comment. I meant should we have different flags for each file, but if `--config` can accept multiple files that should probably be fine.

Reply:
Thanks @coreydaley for clearing out your doubt here. Just wondering Is there any way you are aware of where we can find out whether multiple `--config` are allowed or not? cc_ @gabemontero If you can put on your views on this, that would be really great and help to proceed.

Reply:
It looks like --config can only be used once at the moment, it is just a string var, it would need to be changed to a string slice instead https://github.com/openshift/csi-driver-shared-resource/blob/master/cmd/csidriver/main.go#L31

Reply:
Specifically a StringSliceVar I think

Reply:
I have no additional views @prietyc123 - looks like you and @coreydaley are sorting it out; looks like vanilla golang details from what I can tell

bottom line as long as you get access to the 2 maps for predefined shared secrets and shared config maps, that is what is needed